examples/snmp/apc_ups_temperature_humidity.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * This script can monitor the UPS Temperatury and Humidity.
+ * Communication protocol is SNMP V2
+ * Creates a table with the following columns:
+ *      - Probe Index: The index of the prob
+ *      - Room Temperature: The current temperature reading from the probe
+ *      - Room Humidity: The current humidity reading from the probe in percent
+ */
+
+var iemStatusProbesTableOID = "1.3.6.1.4.1.318.1.1.10.2.3.2";
+var probeIndexOID = "1.3.6.1.4.1.318.1.1.10.2.3.2.1.1.1";
+var probeTempOID = "1.3.6.1.4.1.318.1.1.10.2.3.2.1.4.1";
+var probeTempUnitsOID = "1.3.6.1.4.1.318.1.1.10.2.3.2.1.5.1";
+var probeHumidityOID = "1.3.6.1.4.1.318.1.1.10.2.3.2.1.6.1";
+
+function validateAndGetData() {
+    var d = D.q.defer();
+    D.device.createSNMPSession().walk(iemStatusProbesTableOID, function(out, err) {
+        if (err) {
+            console.error(err);
+            d.failure(D.errorType.GENERIC_ERROR);
+        } else if (!out) {
+            d.failure(D.errorType.RESOURCE_UNAVAILABLE);
+        } else {
+            d.resolve(out);
+        }
+    });
+
+    return d.promise;
+}
+
+/**
+ * @remote_procedure
+ * @label Validate Association
+ * @documentation This procedure is used to validate if the driver can be applied on a device during association as well as validate any credentials provided
+ */
+function validate(){
+    validateAndGetData()
+        .then(function(){
+            D.success();
+        })
+        .catch(function (err) {
+            console.error(err);
+            D.failure(D.errorType.GENERIC_ERROR);
+        });
+}  
+
+/**
+* @remote_procedure
+* @label Get Device Variables
+* @documentation This procedure collects UPS Temperatury and Humidity values and sets it in a table
+*/
+function get_status() {
+    validateAndGetData()
+        .then(function(out) {
```

Comment:
Add validation on the data collected.

This will always create an entry as long as there is a probeNumber response 

Resulting into this:

<img width="812" alt="image" src="https://github.com/domotz/custom-driver/assets/3306549/8601bbe9-9876-41c4-91e7-c62f80cea03e">